pages/common/chmod.md

```diff
@@ -1,36 +1,24 @@
 # chmod
 
-> Change the access permissions of a file or directory.
+> Change file/directory access permissions.
```

Comment:
I think the old description is more explicit.

Reply:
By "more explicit", I mean it has greater clarity.

Reply:
Done 😄